src/sentry/api_gateway/proxy.py

```diff
@@ -47,7 +52,13 @@ def proxy_request(request: Request, org_slug: str) -> StreamingHttpResponse:
     """Take a django request object and proxy it to a remote location given an org_slug"""
     from sentry.types.region import get_region_for_organization
 
-    target_url = get_region_for_organization(None).to_url(request.path)
+    try:
+        region = get_region_for_organization(org_slug)
+    except RegionResolutionError:
+        logger.info("region_resolution_error", extra={"org_slug": org_slug})
+        raise NotFound
```

Comment:
This behaviour is to satisfy an existing test that was written but skipped until `get_region_for_organization` was implemented.